scio-core/src/main/scala/com/spotify/scio/util/Functions.scala

```diff
@@ -150,6 +150,11 @@ private[scio] object Functions {
     override def partitionFor(elem: T, numPartitions: Int): Int = g(elem)
   }
 
+  def simpleFn[T, U](f: T => U): SimpleFunction[T, U] = new SimpleFunction[T, U] {
```

Comment:
Might want to follow suit with `NamedDoFn` and add `private[util] class NamedSimpleFn[T, U] extends SimpleFunction[T, U] with NamedFn` to `NamedFn.scala` and use that instead, so that we get better callsite information.